config/config-example.json

```diff
@@ -1,10 +1,36 @@
 {
-  "adminId": ["111232201848295424", "96335658997526528", "96354827579174912"],
+  "adminId": [
+    "111232201848295424",
+    "96335658997526528",
+    "96354827579174912"
+  ],
   "api": {
     "snipcart": "",
     "steam": ""
   },
-  "color": [248, 154, 34],
+  "applicationId": "",
+  "channels": {
```

Comment:
This is a duplicate, shouldn't be in the config example because it's in the snowflakes example.